src/components/cylc/TaskFilterSelect.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <v-select
+    :label="`Filter by ${type}`"
+    :data-cy="`filter ${type}`"
+    :items="items"
+    clearable
+    multiple
+    :placeholder="`Filter by ${type}`"
```

Comment:
In some cases we want to have the label and in others we want to have the placeholder. So if we don't specify these props they will be [fallthrough attributes](https://vuejs.org/guide/components/attrs.html) and you can specify them on the `<TaskFilerSelect>` tag instead